tests/transpiler/test_conversion_graph.py

```diff
@@ -195,6 +195,28 @@ def test_copy_conversion_graph():
     assert node_alias_id_map_init == copy._node_alias_id_map
 
 
+@pytest.mark.parametrize("program_type", ["qasm2", "qasm3", "pyquil", "braket", "cirq"])
+def test_unregistered_node_in_conversion_graph(program_type):
+    """Test the unregistered nodes in ConversionGraph"""
+    assert program_type in QPROGRAM_ALIASES
```

Comment:
```suggestion
    if program_type not in QPROGRAM_ALIASES:
        pytest.skip(f"{program_type} not installed")
```

`pyquil` is not compatible with Python 3.13, so is not installed during those unit-tests:

https://github.com/qBraid/qBraid/blob/d3f6614b5ea3c58fe0fe1a0a79be68638673e26c/requirements-dev.txt#L6

The change suggested above should fix it.